frontend/src/components/chats/components/chats-list/components/chat/chat.tsx

```diff
@@ -23,12 +25,21 @@ const Chat: FC<Props> = ({
   dateTheLastMessageWasSent,
   onClick,
 }) => {
+  const { currentChatId } = useAppSelector(({ chats }) => ({
+    currentChatId: chats.currentChatId,
+  }));
   const handleChatMessagesLoad = (): void => {
     onClick(chatId);
   };
 
   return (
-    <div className={styles.chat} onClick={handleChatMessagesLoad}>
+    <div
+      className={getValidClasses(
+        styles.chat,
+        currentChatId === chatId && styles.selected,
+      )}
+      onClick={handleChatMessagesLoad}
```

Comment:
`div` is not an interactive element

Reply:
Do I have to put the whole component into button? 😕

Reply:
And Link is not an option as well, because I do not redirect anywhere

Reply:
Yep, button seems like a better option